app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= render "shared/navbar" %>
+<section class="shopping-list-section flex justify-center mt-10">
+    <% if @recipe.present? %>
+        <div class="shopping-list-container w-[40%]">
+            <header class="flex  items-center justify-center">
+                <h2 class="text-2xl font-semibold me-5 "> Shopping Cart</h2>
+                <i class="fas fa-cart-arrow-down text-2xl"></i>
+            </header>
+            <div class="info flex justify-between mt-5">
+                <h2>Amount of food items to buy: <%= @recipe_foods.count %> </h2>
+                <h2>Total value of food needed: $ <%= @total_value %></h2>
+            </div>
+            <div class="table-container mt-10">
+                <table class="border w-full ">
+                    <thead class="border">
+                        <tr>
+                            <th>Food</th>
+                            <th>Quantity</th>
+                            <th>Price</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <% @recipe_foods.each do |recipe_food| %>
+                            <tr>
+                                <th><%= recipe_food.food.name%></th>
+                                <th><%= recipe_food.quantity%></th>
+                                <th><%= recipe_food.total_price%></th>
+                            </tr>
```

Comment:
- [ ] In the General shopping list you should display the food that is missing from all recipes. (Example: You have a food apple with 20 quantities and in a recipe, you added food ingredient apple in 30 quantity means 10 is missing in your food list so you should display the apple food with the price of 10 quantities in the general shopping list). Currently, if any food is added to recipes you're showing the total quantity in the general shopping list. Please make sure to fix this issue.